modules/ardrivo/src/Wire.cxx

```diff
@@ -27,32 +25,32 @@ void TwoWire::begin(int address)
 size_t TwoWire::requestFrom(uint8_t address, size_t size, bool stop)
 {
  
-    ProtocolRequest req = ProtocolRequest(RequestType::REQUEST_SEND, size);
-    transmissionAddress = address;
+    ProtocolRequest req = ProtocolRequest{ REQUEST_SEND, size };
+    transmissionAddress.value() = address;
     
     //auto req = new ProtocolRequest(RequestType::REQUEST_SEND, size);
 
-    board_data->i2c_buses[bus_id].requests.at(address).push_back(req);
+    board_data->i2c_buses[bus_id].requests[address].push_back(req);
 
     size_t bytesReturned = req.bytes;
     return bytesReturned;
 }
 
 uint8_t TwoWire::requestFrom(uint8_t address, uint8_t size)
 {
-    return requestFrom(address, size, static_cast<uint8_t>(true));
+    return requestFrom(address, static_cast<size_t>(size), static_cast<bool>(true));
```

Comment:
What's the point of casting `true` to a `bool`?
Also, widening conversions from `uint8_t` can simply be performed by a simple unary `+`, i.e. `+size`